client/fresheyes-web/src/app/components/modal/SuccessModal.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { useEffect } from "react";
+import Image from "next/image";
+import checkmarkIcon from "../../assets/checkmark.svg";
+import { usePathname, useSearchParams } from "next/navigation";
+
+export const SuccessModal = () => {
+  const pathname = usePathname();
+  const [isOpen, setIsOpen] = React.useState(false);
+  const params = useSearchParams();
+
+  useEffect(() => {
+    const isInstalled = params.get("installation_id") !== null;
+    if (isInstalled) {
+      setIsOpen(true);
+    }
+  }, [pathname]);
+
+  const removeModal = () => {
+    setIsOpen(false);
+    let url = new URL(window.location.href);
```

Comment:
We can use the `useRouter` from `next/navigation` `router.push` here. It saves us from depending on the window object which throws a warning in the console if undefined. We also remove one line of code

Reply:
Alright, will make changes using `useRouter`